### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by freddieline**
Added a step for non-Divido tenants to initialize the merchant sdk with a custom `base_uri`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,31 @@ $httpClientWrapper = new \Divido\MerchantSDK\HttpClient\HttpClientWrapper(
...
+$identifier = strtoupper($array[0]);
+$env =  ('LIVE' == $identifier)
+    ? constant("Divido\MerchantSDK\Environment::PRODUCTION")
+    : constant("Divido\MerchantSDK\Environment::$identifier");
+
+// create a client wrapper
+$client = new \GuzzleHttp\Client();
+$httpClientWrapper = new \Divido\MerchantSDK\HttpClient\HttpClientWrapper(
+    new \Divido\MerchantSDKGuzzle6\GuzzleAdapter($client),
+    "https://merchant-api-pub.sandbox.example-tenant.divido.net/",
```
Comment (andivido) [inline README.md:47]: Might be more readable to use TENANT_URI and TENANT_API_KEY then have a blurb before the code explaining that you should be supplied these details by Divido

**Review by andivido — COMMENTED** [review]
Looks good. Just thought I'd make a suggestion for readability. Take it or leave it!

File: README.md (hunk 1 above)
Comment (freddieline) [reply README.md]: Good point! Done.

**Review by andivido — DISMISSED** [review]
👍

**Comment by freddieline** [comment]
The was a phpstan error too which is sorted now.
```
Error: Variable $segment in isset() always exists and is not nullable.
```
This fixed by removing the redundant `isset($segment)` condition

**Review by andivido — APPROVED** [review]

**Pull request merged**